packages/node_modules/samples/browser-call-with-screenshare/app.js

```diff
@@ -173,9 +173,9 @@ function bindMeetingEvents(meeting) {
   });
 
   // Of course, we'd also like to be able to end the meeting:
-  document.getElementById('hangup').addEventListener('click', () => {
-    meeting.leave();
-  });
+  const leaveMeeting = () => meeting.leave();
```

Comment:
Nice find but this seems out of scope for this PR. Feel free to move to a different PR or even a separate commit in this same PR with `fix(samples)` commit message.